vertical-pod-autoscaler/README.md

```diff
@@ -210,6 +212,17 @@ kubectl delete clusterrolebinding myname-cluster-admin-binding
   using VPA together with [Cluster Autoscaler](https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/FAQ.md#basics).
 * Multiple VPA resources matching the same pod have undefined behavior.
 
+## Limitations of the beta version
+
+The limitations from alpha version still apply except for note about `Auto`
```

Comment:
It's unclear to me which note you mean. Anyway, I will be merging these two sections anyway, so maybe not that important.

Reply:
Oh, I see this is mentioned below. You can disregard this comment then.